tests/providers/openlineage/plugins/test_openlineage_adapter.py

```diff
@@ -206,6 +296,63 @@ def test_emit_complete_event(mock_stats_incr, mock_stats_timer):
     mock_stats_timer.assert_called_with("ol.emit.attempts")
 
 
+@mock.patch("airflow.providers.openlineage.plugins.adapter.Stats.timer")
+@mock.patch("airflow.providers.openlineage.plugins.adapter.Stats.incr")
+def test_emit_complete_event_with_additional_information(mock_stats_incr, mock_stats_timer):
+    client = MagicMock()
+    adapter = OpenLineageAdapter(client)
+
+    run_id = str(uuid.uuid4())
+    event_time = datetime.datetime.now().isoformat()
+    adapter.complete_task(
+        run_id=run_id,
+        end_time=event_time,
+        parent_job_name="parent_job_name",
+        parent_run_id="parent_run_id",
+        job_name="job",
+        task=OperatorLineage(
+            inputs=[Dataset(namespace="bigquery", name="a.b.c"), Dataset(namespace="bigquery", name="x.y.z")],
+            outputs=[Dataset(namespace="gs://bucket", name="exported_folder")],
+            job_facets={"sql3": SqlJobFacet(query="SELECT 3;")},
+            run_facets={"sql1": SqlJobFacet(query="SELECT 1;")},
+        ),
+    )
+
+    assert (
+        call(
+            RunEvent(
+                eventType=RunState.COMPLETE,
+                eventTime=event_time,
+                run=Run(
+                    runId=run_id,
+                    facets={
+                        "parent": ParentRunFacet(
+                            run={"runId": "parent_run_id"},
+                            job={"namespace": "default", "name": "parent_job_name"},
+                        ),
+                        "parentRun": ParentRunFacet(
+                            run={"runId": "parent_run_id"},
+                            job={"namespace": "default", "name": "parent_job_name"},
+                        ),
+                        "sql1": SqlJobFacet(query="SELECT 1;"),
```

Comment:
The same question 🙂

Reply:
Let's use the above topic to discuss this, as this is the same case.